src/server.py

```diff
@@ -1,35 +1,39 @@
-import os
 import asyncio
+
 import websockets
+
 from cuda_utils import cuda_device_enable
-from model_operations import load_model, generate_tokens
+from model_operations import generate_tokens, load_model
 
 # Ensure the model is loaded on the GPU
 print(f"device set to: {cuda_device_enable()}")
 
 # Load the model and tokenizer
-model_name = os.environ['MODEL_REPO']
-model_file = os.environ['MODEL_FILE']
+model_name = "TheBloke/dolphin-2.0-mistral-7B-GGUF"
+model_file = "dolphin-2.0-mistral-7b.Q4_K_M.gguf"
```

Comment:
We removing the env var for hard-coded values?

Reply:
Done